### DISCUSSION TITLE
Welcome to propka Discussions!

### DISCUSSION THREAD

**Discussion by sobolevnrm** [jensengroup/propka#116] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by intendo** [comment]
I am @intendo and I have been a computer scientist for about 35 years. I like to help scientists make their code cleaner, smaller, faster, well tested, and production ready.
